server/data/__testutils/testObjects.ts

```diff
@@ -67,3 +67,22 @@ export const createSupportTypes = ({
     },
   ] as SupportType[],
 } = {}): SupportType[] => supportTypes
+
+export const createPrisonerProfile = ({
```

Comment:
Just wondering about the name and whether `createPrisonerProfile` risks being confused with the specific `PrisonerProfile` type we have ([here](https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/9aded01a1d26dafcd31f7f0bda59f5594ebfe179/server/%40types/bapv.d.ts#L122)) that's used to pass all the data for the Prisoner Profile page.

Perhaps something like just `createPrisoner`, `createInmate` or just be explicit with `createInmateDetail`? Tricky I know because we've a lot of overlapping terms!